tests/conftest.py

```diff
@@ -14,7 +34,8 @@ def testing_images():
     async def _pull():
         docker = Docker()
         required_images = [
-            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine'
+            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine',
+            'busybox:latest',
```

Comment:
Actually, do you have a preference for alpine over busybox btw? I think it's really the same in the end.